migrations/0030-add-order-status-field.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner, TableColumn, TableIndex } from 'typeorm';
+
+const TABLE_NAME = 'Orders';
+
+export class AddOrderStatusField1633030219180 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn(TABLE_NAME, new TableColumn({
+      name: 'status',
+      type: 'varchar(255)',
+      default: '\'PLACED\'',
```

Comment:
![image](https://user-images.githubusercontent.com/33337209/139558226-ba02b323-950b-4082-8ae8-051f044855e6.png)

The migration fails without the quotation marks, because PG thinks it's a column otherwise